spec/support/user_helper.rb

```diff
@@ -1,5 +1,6 @@
 def login(email)
   visit root_path
+  click_link 'Войти'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.